mappings/UofSC.py

```diff
@@ -12,7 +12,7 @@ def createMapping(self):
             'authors': [('authors', '{0}|||true')],
             'dates': [('publicationDate', '{0}|publication_date')],
             'publisher': [('publisher', '{0}')],
-            'spatial': ('publisherLocation', '{0}|publisherLocation'),
+            'spatial': ('publisherLocation', '{0}'),
```

Comment:
is this and the ISAC change below meant to be in this PR?

Reply:
Yes it is. This is just a typo that both mappings had that I wanted to get rid of now.